package.json

```diff
@@ -19,55 +19,50 @@
     "lint:spec:oas": "spectral lint src/specs/api.yaml",
     "lint:spec": "yarn lint:spec:oas",
     "audit:fix": "yarn-audit-fix",
-    "build": "yarn clean && webpack",
-    "clean": "rimraf dist coverage",
-    "package:sam": "sam build -t src/infra/main/template.yaml",
+    "clean": "rimraf .aws-sam coverage",
+    "build": "sam build -t src/infra/main/template.yaml",
     "package:sam:fix": "run-script-os",
```

Comment:
are these scripts actually needed anymore or can we loose package:sam:fix?

Reply:
I have tested it a few times and I cannot see that it is changing anything in the template file.. I will remove it and test again that everything builds fine

Reply:
done. Also removed deps that was used to parse the yaml